### ISSUE TITLE
The current ffmpeg version does not support animated webp

### ISSUE THREAD

**Issue by kaya151** [a-schild/jave2#253]
The current ffmpeg version does not support animated webp. Is it possible to add ffmpeg 6.1?

**Comment by a-schild** [comment]
I have not yet tested it with ffmpeg 5 or 6.
But you could do this yourself, just implement your own DefaultFFMPEGLocator which then uses the 6.x version.
If it passes all tests, then we could integrate the 6.1 versions in jave

**Comment by rob93c** [comment]
Hi @a-schild, I went through FFmpeg's tickets and I found out they don't support this feature as of now (not even with their latest release).

Ticket reference: https://trac.ffmpeg.org/ticket/4907

**Comment by a-schild** [comment]
Yes, and further than asked. 4.0.0 bundles **ffmpeg 9.0.1**, not 6.1.

Animated webp is there and works. The encoder:

```
V....D libwebp_anim    libwebp WebP image (codec webp)
```

and an actual encode against the shipped binary produced a 147 kB animated file:

```java
VideoAttributes video = new VideoAttributes();
video.setCodec("libwebp_anim");
```
